ashutosh-rath02/git-re
# Fix: Testimonial link on Leaderboard page does not navigate to the Testimonial page

## Description
This pull request addresses the issue where the link to the Testimonial page on the Leaderboard page does not navigate to the Testimonial page. This fix ensures that users can access the Testimonial page by clicking the link on the Leaderboard page.

### Changes Made
-  Corrected the `href` attribute for the Testimonial link on the Leaderboard page to ensure proper navigation.

### Issue Reference
Fixes [#287 ] 

### Steps to Test
1. Go to the Leaderboard page.
2. Click on the Testimonial link.
3. Verify that the page navigates to the Testimonial page.

### Expected Behavior
After clicking the link to the Testimonial page on the Leaderboard page, the user should be redirected to the Testimonial page.

### Screenshots
![image](https://github.com/ashutosh-rath02/git-re/assets/76616023/b02aa449-9d34-4104-a7b5-8d1a3300e4ef)
